verify-matching-service-test-tool/publish.gradle

```diff
@@ -0,0 +1,20 @@
+apply plugin: 'maven-publish'
+
+def build_number = System.getProperty('IDA_MSA_VERSION')
```

Comment:
`IDA_MSA_VERSION` relies on that env var specifically existing (which we will likely rename in Jenkins 2). `BUILD_NUMBER` is a Jenkins built in that we can always rely on being there (and since we are unlikely to ever set the app_version != build_number I think this is safe)